src/app/pages/cart/components/cart-item/cart-item.component.ts

```diff
@@ -26,7 +26,7 @@ export class CartItemComponent implements OnInit {
   private readonly destroyRef = inject(DestroyRef);
 
   @HostBinding('class')
-  private readonly classes = 'flex flex-col md:flex-row items-center md:justify-around gap-3 w-full md:w-10/12 active:outline-0 py-4 my-4 md:h-72 rounded-md border-2 border-grey-300 shadow-md hover:shadow-xl hover:shadow-zinc-900 duration-300';
+  private readonly classes = 'flex flex-col md:flex-row items-center md:justify-around gap-3 w-full md:w-10/12 active:outline-0 py-4 my-4 md:h-72 rounded-md border-2 border-grey-300 shadow-md hover:shadow-xl duration-300';
 
   constructor(private cartFacadeService: CartFacadeService, private translate: TranslateService) {}
```

Comment:
maybe 'translateService' would be better name for 'translate'

Reply:
@WeszeRita It's optional, sometimes people name it as `translate` and sometimes `translateService`, it's an exception.
The same exception we can make for facade services, for example. It's also okay to name facades as `someFacadeService` or `someFacade`.